File: app/src/main/java/kz/shymkent/relaxhouse/PushNotification.java
package kz.shymkent.relaxhouse;

import android.util.Log;

import com.onesignal.OneSignal;

import org.json.JSONException;
import org.json.JSONObject;

public class PushNotification  {
    public static void createPushNotification(String text){
        try {
            OneSignal.postNotification(new JSONObject("{'contents': {'en':'"+text+"'}, 'include_player_ids': [" + "'"+ Constants.COTTAGE_OWNER_PLAYER_ID+"','"+Constants.COTTAGE_ADMINISTRATOR_PLAYER_ID+"','"+Constants.COTTAGE_CLEANER_PLAYER_ID+"']}"),
                    new OneSignal.PostNotificationResponseHandler() {
                        @Override
                        public void onSuccess(JSONObject response) {
                            Log.i("OneSignalExample", "postNotification Success: " + response.toString());
                        }
                        @Override
                        public void onFailure(JSONObject response) {
                            Log.e("OneSignalExample", "postNotification Failure: " + response.toString());
                        }
                    });
        } catch (JSONException e) {
            e.printStackTrace();
        }
    }
}

File: app/src/main/java/kz/shymkent/relaxhouse/PushNotification.kt
package kz.shymkent.relaxhouse

import android.util.Log
import com.onesignal.OneSignal
import org.json.JSONObject
import com.onesignal.OneSignal.PostNotificationResponseHandler
import kz.shymkent.relaxhouse.mainActivity.MainActivityViewModel.Companion.playerIdList
import org.json.JSONException

object PushNotification {
    @JvmStatic
    fun createPushNotification(text: String) {
        try {
            OneSignal.postNotification(JSONObject("{'contents': {'en':'" + text + "'}, 'include_player_ids': [" + "'" + playerIdList[0] + "','" + playerIdList[1] + "','" + playerIdList[2] + "']}"),
                object : PostNotificationResponseHandler {
                    override fun onSuccess(response: JSONObject) {
                        Log.i("OneSignalExample", "postNotification Success: $response")
                    }

                    override fun onFailure(response: JSONObject) {
                        Log.e("OneSignalExample", "postNotification Failure: $response")
                    }
                })
        } catch (e: JSONException) {
            e.printStackTrace()
        }
    }

}